### ISSUE TITLE
Add support for `/*  */` comments under the comments rule

### ISSUE THREAD

**Issue by najela** [brigade/scss-lint#549]
Presently, under the comments rule, I just have  

```
  Comment:
    enabled: true
```

I'll like to Enforce the use of `/* */` comments everywhere

**Comment by sds** [comment]
Hey @najela,

I'd be open to a pull request adding support for a `style` option to allow you to choose which comment style to enforce, but can you give me some idea of why you want to enforce `/* */`? Those kinds of comments are [actually rendered in your CSS](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#comments), which increases the size of your assets (thus resulting in a slower page load). 

In other words, I think this might be a Bad Idea™, but I'd love to hear about your use case so I can better understand. Thanks!

**Comment by najela** [comment]
@sds Thats for the explanation. I'm just used to the habit of using `/* */` comments when writing code so I wanted to use the same commenting habit in my scss files

**Comment by sds** [comment]
Added a `style` option in 276668aa077d4f69fa1997ee2c54500a8c967048.

**Comment by najela** [comment]
Thanks :+1:
